search-app/docs/api.md

```diff
@@ -1569,7 +1571,13 @@ Lines are provided as a list of comma separated values representing coordinates
 
     curl "%CMR-ENDPOINT%/collections?line=-0.37,-14.07,4.75,1.27,25.13,-15.51"
 
-Note: A query could consist of multiple spatial parameters of different types, two bounding boxes and a polygon for example. If multiple spatial parameters are present, all the parameters irrespective of their type are AND'd in a query. So, if a query contains two bounding boxes and a polygon for example, it will return only those collections which intersect both the bounding boxes and the polygon.
+##### <a name="c-wkt"></a> Wkt
+
+Various spatial geometries can be represented using a subset of Well Known Text (WKT). The currently supported geometries include: point, line, and polygon.
```

Comment:
It may be worth adding some more detail about the expected formatting of polygons and lines (i.e. which point in a polygon corresponds to it's respective value in the supplied parameters)